eZ/Bundle/EzPublishCoreBundle/Command/RemoveOrphanRecordsFromEzUserTablesCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * File containing the DeleteUsersFromTrashCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\Core\Repository\Values\ContentType\ContentType;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use eZ\Publish\API\Repository\Values\Content\Query;
+
+class RemoveOrphanRecordsFromEzUserTablesCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ezpublish:removeorphanrecordsfromezusertables')
```

Comment:
I think we need to find a more namespaced name. Also, words should be separated by hyphens.

Something like `ezplatform:upgrade:remove-orphan-user-records` ?